notes/documents-received-testing.vbs

```diff
@@ -0,0 +1,5260 @@
+								Else 
+									pben_row = pben_row + 1
+								End If
+							Else 
+								pben_row = pben_row + 1
+							End If
+						Loop Until pben_row = 14
+						If pben_row = 14 and pben_updated <> TRUE Then  				'If all lines on the panel are full then it cannot update PBEN
+							MsgBox "PBEN panel is full in " & MAXIS_footer_month & "/" & MAXIS_footer_year & ". Script cannot updated PBEN automatically. Manually update it after script run."
+							iaa_update_pben_checkbox = unchecked
+						End If
+					End If
+				Next
+			
+			ElseIf iaa_ssi_form_received_checkbox = checked AND iaa_form_received_checkbox = unchecked Then 
+
+				orginal_iaa_benefit = iaa_benefit_type
+				iaa_benefit_type = "02-SSI"
+				If iaa_date_applied_pben = "" and iaa_benefit_type = "02-SSI" then iaa_disposition_code_dropdown = "N"	'Handling for error in maxis
+
+				For each thing in date_array
+					pben_disp_code_string = ""	'reset to blank for each month 
+					pben_month_already_updated = FALSE	'set to false until verified that the exact same line exists
+					MAXIS_footer_month = datepart("m", thing)
+					If len(MAXIS_footer_month) = 1 Then MAXIS_footer_month = "0" & MAXIS_footer_month 
+					MAXIS_footer_year = right(datepart("yyyy", thing), 2)
+					Do
+						Call Navigate_to_MAXIS_screen ("STAT", "PBEN")					'Go to PBEN 
+						EMReadScreen nav_check, 4, 2, 49
+						EMWaitReady 0, 0
+					Loop until nav_check = "PBEN"
+									
+					Call write_value_and_transmit(pben_member_number, 20, 76)			'Go to the correct member 
+					
+					'Read pben lines to see if pben month already updated
+					EMReadScreen pben_benefit_line_1, 2, 8, 24
+					EMReadScreen pben_benefit_line_2, 2, 9, 24
+					EMReadScreen pben_benefit_line_3, 2, 10, 24
+					EMReadScreen pben_benefit_line_4, 2, 11, 24
+					EMReadScreen pben_benefit_line_5, 2, 12, 24
+					EMReadScreen pben_benefit_line_6, 2, 13, 24
+					pben_benefit_line_1= replace(pben_benefit_line_1, "_", "")
+					pben_benefit_line_2= replace(pben_benefit_line_2, "_", "")
+					pben_benefit_line_3= replace(pben_benefit_line_3, "_", "")
+					pben_benefit_line_4= replace(pben_benefit_line_4, "_", "")
+					pben_benefit_line_5= replace(pben_benefit_line_5, "_", "")
+					pben_benefit_line_6= replace(pben_benefit_line_6, "_", "")
+					pben_benefit_line_1 = cstr(pben_benefit_line_1)
+					pben_benefit_line_2 = cstr(pben_benefit_line_2)
+					pben_benefit_line_3 = cstr(pben_benefit_line_3)
+					pben_benefit_line_4 = cstr(pben_benefit_line_4)
+					pben_benefit_line_5 = cstr(pben_benefit_line_5)
+					pben_benefit_line_6 = cstr(pben_benefit_line_6)
+
+					EMReadScreen pben_line_1, 54, 8, 40
+					EMReadScreen pben_line_2, 54, 9, 40
+					EMReadScreen pben_line_3, 54, 10, 40
+					EMReadScreen pben_line_4, 54, 11, 40
+					EMReadScreen pben_line_5, 54, 12, 40
+					EMReadScreen pben_line_6, 54, 13, 40
+					pben_line_1 = trim(replace(pben_line_1, "_", ""))
+					pben_line_2 = trim(replace(pben_line_2, "_"
```

Comment:
4764: the_asset isn't defined for the array before this point, needs to be indexed? or move this down into the next For the_asset loop